data/card_catalogue/unique-1.json

```diff
@@ -264,37 +329,49 @@
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "Targeted NPC becomes stuck in place for ten seconds.",
+      "rules": {
+        "active": "Choose an NPC. They are stuck in place for ten seconds.",
+        "passive": ""
+      },
       "flavour": "Have my legs always been this heavy?",
       "image": "assets/core/card/illustration/Locomotor Hex.png"
     }
   },
   {
-    "title": "Rime Charm",
+    "title": "Rime",
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Cover a non-magic object in a thin layer of ice.",
+      "rules": {
+        "active": "Cover an object in a thin layer of ice.",
+        "passive": ""
+      },
       "flavour": "Whoa, chill.",
-      "image": "assets/core/card/illustration/Rime Charm.png"
+      "image": "assets/core/card/illustration/Rime.png"
     }
   },
   {
-    "title": "Adjoint Immolation Hex",
+    "title": "Akindle",
```

Comment:
This one was also due a new name imo, not sure this new one's all that great though